\chapter{Applying N-of-1-\emph{pathways} MD to\\analyze single-cell RNA-sequencing} \label{Chap:ctcs}

\indent \indent With an aim to discover what other insights the paired statistics resulting from N-of-1-\emph{pathways} MD could obtain, we modified the framework to analyze single-cell RNA-sequencing (scRNA-seq) data. Each cell in an organism is unique and aggregating single-cell signal across many cells dilutes and muddies the picture obtained from scRNA-seq. The development of paired, mechanism-anchored statistics between cells lends insight into cell-cell heterogeneity that is obfuscated by other methods and mitigates sample size requirements of cohort-based statistics. I developed a new framework to analyze single-cell transcriptomes from circulating tumor cells (CTCs) isolated from blood samples obtained from prostate cancer patients (Appendix \ref{App:E}; \cite{Patel2014,Schissler2016}). By aggregating the cell-cell MD pathway scores, therapeutic-resistance mechanisms were identified from only thirteen patients (five drug-resistant and eight drug-na\"{\i}ve). Further, novel visualizations of cell-cell pathway heterogeneity were developed by modifying wind rose plots (\cite{Court1963}). This project illustrated the flexibility of these methods outside the scope of \emph{single-subject inference} to the arena of small sample cohort sizes, enabling the study of rare diseases and smaller scale, limited-resource projects.

Again, the development of this article was a joint work. I proposed the study and, along with Dr. YA Lussier, conceived the notion of computing pathway scores for every pair of cells. I developed the three perspectives obtainable from different aggregation approaches. Myself and Dr. WW Piegorsch developed the ``clustered'' bootstrap to account for correlation between cells derived from the same patient. Dr. YA and I designed the rose plots. Finally, I authored the first draft.
