\chapter{Applying N-of-1-\emph{pathways} MD to analyze single-cell RNA-sequencing} \label{Chap:ctcs}

With an aim to discover what other insights these paired statistics could provide in situations where conventional methods fail, we modified the N-of-1-\emph{pathways} MD framework to analyze single-cell RNA-sequencing (scRNA-seq) data. Each cell in an organism is unique and aggregating single-cell signal across many cells dilutes and muddies the picture obtained from scRNA-seq. The development of paired, mechanism-anchored statistics between cells lends insight into cell-cell heterogeneity that is obfuscated by other methods and mitigates sample size requirements of cohort-based statistics. We developed a new framework to analyze single-cell transcriptomes from circulating tumor cells (CTCs) isolated from blood samples obtained from prostate cancer patients (\cite{Patel2014,Schissler2016}). By aggregating the cell-cell MD pathway scores, therapeutic-resistance mechanisms were identified from only 13 patients (5 drug-resistant and 8 drug-na\"{\i}ve). Further, we developed novel visualizations of cell-cell pathway heterogeneity by modifying wind rose plots(\cite{Court1963}). This project illustrated the flexibility of these methods outside the scope of \emph{single-subject inference} to the arena of small sample cohort sizes, enabling the study of rare diseases and smaller scale, limited-resource projects. 

- refer to Appendix X \\
- critical picture\\
- my contribution to the project, mention last table as precursor to CTCs\\ 
- contribution to the field\\
