\chapter{Introduction}\label{Chap:Intro}

\section{Article-based dissertation and organization}\label{sec:org}

format of this document
unaltered published format

\section{Single-subject transcriptome analytics}\label{sec:nof1}

\indent \indent The bulk of the projects described below involve the development of single-subject transcriptome (i.e. gene expression data) methodology for precision medicine. The motivation for single-subject biomedical methods is straightforward. Biology is exceeding complex - each patient is inherently unique. Demographics, genetics, and epi-genetics (to name a few) all contribute to a patient's diagnosis, treatment, and prognosis. Often a patient is given a drug that has, on average, a high success rate for their medical condition, but inexplicably the patient fails to respond and suffers a poor outcome. What if we can make treatment decisions and predictions for that \emph{individual's} dysregulated cellular mechanisms? 

The applied problem of single-subject studies is intriguing, but as statisticians we are trained to make inferences on population parameters, based on (often independently) sampled observations. Numerous philosophical and technical issues arise when dealing with only a single patient's data. (It is worth noting that Fisher's famous tea-tasting experiment can be viewed as a \emph{single-subject} study.) That being said, applications for statistical \emph{single-subject inference} have created a demand for developing this unexplored area. From a statistical research standpoint, \emph{single-subject inference} poses many open problems that involve statistical topics including multivariate statistics, computing, machine learning, high-dimensional data analysis, small sample analysis, paired-sample statistics, big data, and data visualization.

\section{N-of-1-\emph{pathways} framework}\label{sec:nof1pathways}

The first project of my dissertation research concerned the ``statistical component'' of the single-subject framework, N-of-1-\emph{pathways} developed in the lab which I work (\cite{Gardeux2014}). The basic goal is to discover single-subject dysregulated genetic pathways from gene expression data. Two paired samples are obtained from a patient, one from a baseline and one from a case condition (e.g., blood before and after treatment, tumor and normal tissue, etc.). The messenger-RNA (mRNA) expression is then quantified from the samples (via RNA-sequencing or microarrays). Next the workflow takes an informatic turn - the mRNAs are annotated to gene sets (pathways) that are functionally related via knowledgebases, such as Gene Ontology (\cite{Ashburner2000}). This is in effect a dimension-reduction technique that also provides mechanistic interpretation (indeed single biomarkers, mRNA or otherwise, have been notoriously difficult to use in creating therapies or classifiers). The last step in the framework is the statistical component. The differential expression within each pathway is quantified and a test of hypotheses is conducted. We coined the phrase \emph{differentially expressed pathway} (DEP) to describe when a null hypothesis of equal expression is rejected (as an analog to the well-known DEG; differentially expressed gene). 

