\chapter{Concluding remarks}\label{Chap:conclusion}

\section{Summary}
\indent\indent The final chapter of this dissertation serves as a summary of the work presented in the previous chapters. My goal is to become a pioneer and leader in the field of \emph{single-subject inference}, in which there are many open questions for further study. There is also substantial demand for these methods in the biomedical domain, within which I am passionate to make contributions. I plan to focus my future research in these areas as I develop my academic career.

\section{Other ongoing works}
I am involved in several other projects that focus on refining, applying, and disseminating our methods. In brief, one project is working to improve the clustering step of kMEn (Section \ref{sec:kmen}; \cite{Li2016}). Another project is developing new methodology to predict asthma exacerbation by coupling our clinical assay virogram (\cite{Gardeux2015}) with N-of-1-\emph{pathways}. Further, we are releasing an \texttt{R} package for N-of-1-\emph{pathways} and the cell-cell distance analysis, and plan to submit a short application note describing its utility.

\section{Two proposed projects and future work}

The first proposed project expands my recently submitted article accounting for inter-gene correlation (Section \ref{sec:intergene}). This previously developed method takes a frequenist approach while also injecting additional external information into the N-of-1-\emph{pathways} framework. It is natural when considering the incorporation of external information, however, to consider some form of Bayesian methodology (\cite{Christensen11}). Indeed, the inter-gene correlation structure could be modeled in a hierarchical fashion - providing a rich setting for differential pathway metric estimation and testing. Further still, a whole-transcriptome gene set testing approach in this Bayesian hierarchical model could be devised to ``borrow strength'' across pathways. (One could view genetic pathways as a spatially-correlated regions informed by the gene set ontological, topological, structure). However, this framework would likely be highly complex and, with the limited data on hand (two paired samples from one subject), subjective (prior) information and assumptions may heavily dominate the eventual inferences. Still, the flexibility, robustness, and predictive performance of a Bayesian framework shows promise to provide an effective tool for precision medicine.

A second project would enhance the methodology presented in the aggregated cell-cell statistical distances article (Section \ref{sec:ctcs}; \cite{Schissler2016}). Briefly, the original framework includes calculating pathway differential expression between each pair of cells. Then a nonparametric, hierarchical bootstrap (a ``clustered'' bootstrap) was conducted to assess whether a pathway was differentially expressed between two groups of cells, sampled from patients. Two main issues could be explored here: First, single-cell RNA-sequencing is known to have much higher dropout rates (mRNA transcripts not amplified) than bulk RNA-seq. A simulation study of the effect of zero-inflation on pathway testing would be instrumental in determining the empirical pathway testing operating characteristics (size and power) and possibly lead to modifications of the framework (including modeling of dropout and amplified mRNA through mixture modeling). Second, by construction, the metrics generated from pairing all the cells are correlated. The underlying, induced covariance structure should be studied and potentially modeled to determine the multivariate characteristics of the pathway cell-cell distances. This could lead to interesting statistical insights and novel significance tests in the small-sample setting.

More far-sighted projects could include extending our transcriptome analytics to proteomics and to multiple samples (as opposed to simply paired). The transcriptome offers a glimpse at actual protein molecule activity within the cell, yet opinions differ on the correlation between the mRNA expression and the protein activity. Proteomics represent an ability to quantify protein abundance and states, but remains technically challenging and expensive. As proteomics improve, in principle, many of our single-subject, gene-expression-based methods could be easily modified for detecting protein-level pathway differences. One could also develop analytics based on multiple samples from a single-patient. Hearkening back to classical N-of-1 studies in which a single measurement is tracked over time (\cite{Nikles2015}), we could investigate single-subject, temporal pathway expression changes to make adaptive treatment decisions (one can envision some form of time-series approach). Or, for spatially-structured multiple samples (e.g., multiple samples from a tumor), we could generalize our distances metrics for higher dimensions and/or model the spatial auto-correlation.
