%\documentclass[draft]{ua-thesis}
\documentclass[final]{ua-thesis}
\usepackage{multirow}
\usepackage[section] {placeins}
\usepackage{setspace}
%\onehalfspacing
%\doublespacing

% this package allows insertion of entire pdfs
\usepackage[final]{pdfpages}

%\linespread{1.75}
%\addtolength{\footskip}{24 pt}  % pushes page no. down at bottom of page

%\usepackage[breaklinks]{hyperref}
\usepackage{graphicx}
\usepackage{amsmath,amsfonts,amssymb}
\usepackage{amsthm}
%\usepackage{soul}  %for underline characters
%\usepackage{natbib}
%\bibliographystyle{plainnat}
%\bibpunct{(}{)}{;}{a}{,}{,}
\usepackage[backend=biber,style=authoryear,citestyle=authoryear,sorting=none]{biblatex}
\bibliography{/Users/grantschissler/Dropbox/bib/default,/Users/grantschissler/Dropbox/bib/library}
\defbibheading{refs}[refs]{%
\chapter*{References}%
}
\DeclareSourcemap{
  \maps[datatype=bibtex, overwrite]{
    \map{
      \step[fieldset=url, null]
    }
  }
}


\newtheorem{definition}{Definition}[section] %For definition and numbered in sections
\numberwithin{equation}{section}
%\numberwithin{table}{subsection}
%\numberwithin{figure}{subsection}

%\long\def\symbolfootnote[#1]#2{\begingroup% For special symbols in footnote
%\def\thefootnote{\fnsymbol{footnote}}\footnote[#1]{#2}\endgroup}

%\setlength{\topmargin}{0in}
%\setlength{\textheight}{8.5in}
%\setlength{\evensidemargin}{0in}
%\setlength{\oddsidemargin}{0in}
%\setlength{\textwidth}{6.5in}

\newcommand\solidrule[1][1cm]{\rule[0.5ex]{#1}{.4pt}}
\newcommand\dottedrule{\mbox{%
\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}%
\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}}}
\newcommand\dashedrule{\mbox{%
\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}}}
\newcommand\dasheddotted{\mbox{%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}}}

\director{Walter W. Piegorsch \& Yves A. Lussier}
\author{A. Grant Schissler}
\title{Statistical and Bioinformatic Contributions to Single-Subject Transcriptome Analytics}
\date{2016}
    \makeindex

%\ifpdf \pdfinfo{ /Author (Qijun Fang) /Title  (Hierarchical Bayesian Benchmark Dose Analysis) } \fi


\begin{document}
\maketitle

\chapter*{Acknowledgments}
First and foremost, I would like to express my heartfelt gratitude towards my co-advisors, Dr. Walter W Piegorsch and Dr. Yves A Lussier. Both have invested countless hours in my training and provided me with unique tools, experience, and perspectives. From Yves, I learned what it takes to produce rapidly in an academic environment. His knowledge, determination, creativity, and commitment to biomedical science is inspiring. From Walt, I am finally beginning to ``think like a statistician'', a gift which was my motivation for this graduate experience. His ability to explain statistical ideas through his writing and presentation is truly amazing. His creativity, curiosity, and demeanor are uplifting and motivating. It hard to put into words how fortunate I feel to have worked alongside these two men.

Secondly, thanks to Dr. Joe Watkins and the rest of Interdisciplinary Program in Statistics admissions committee for giving me this opportunity. Not many programs would see the potential in a non-traditional student coming from a role as high school stats teacher. All the faculty in the GIDP have supported me from the beginning. From Dr. Lingling An whom connected me with industry working at HTG Molecular with Dr. James Li (who taught me how to \emph{really} code in R) to Dr. Helen Zhang and Dr. Dean Billheimer whom have taught and collaborated with me on these projects. A special thank you to Dr. Ryan Gutenkunst and Dr. Joe Watkins for serving on my committee and providing invaluable guidance.

Third, I would like to thank the entire Lussier Group. Over the past two plus years, everyone of you have influenced me and helped me countless times. Thank you Dr. Colleen Kenost for your support and guidance. Thank you Dr. Ikbel Achour for teaching me write scientific articles (``tell a story'') and Drs. Vincent Gardeux and Haiquan Li for their computational expertise and guidance. And to Mr. Qike Li, my fellow Stats GIDP student, I cannot express how helpful your discussions have been. Being able to take risks and talk freely with you has given me great confidence and knowledge. Of course, it would be remiss if I didn't thank Yves for giving me the opportunity in the Group - this dissertation would not exist without you.

Briefly, thank you to Mr. Larry Kull. Without you providing me a teaching opportunity, I would not have been able to leave my job to return to graduate school. Your humor, kindness, and dedication to students of all backgrounds is an inspiration.

Finally, I would like to thank my wife Marnell for her love and support. I hope that putting up with disrupted sleep and general craziness were worth it. You remain the best thing that has ever happened to me and I hope I can make you proud as we continue our lives together. Thank you to my family - Mom, Dad, Kara, Donald, Allie, Samuel, Brian, Kathy, Brianne, and Eric - for all their encouragement and support (as well as my extended family). Also, thank you to my beloved pets for their unconditional love and reminding me to enjoy the present moment.

\chapter*{Dedication}
\thispagestyle{topright}
\begin{center}In loving memory of my father, Mr. Patrick A. Schissler.\end{center}

%\begin{vim_bug_workaround}
%\end{vim_bug_workaround}

\tableofcontents

\listoffigures
\listoftables

\chapter*{Abstract}
\noindent This dissertation serves as a unifying document for three related articles developed during my dissertation research. The projects involve the development of single-subject transcriptome (i.e. gene expression data) methodology for precision medicine and related applications. Traditional statistical approaches are largely unavailable in this setting due to prohibitive sample size and lack of independent replication. This leads one to rely on informatic devices including knowledgebase integration (e.g., gene set annotations) and external data sources (e.g., estimation of inter-gene correlation). Common statistical themes include multivariate statistics (such as Mahalanobis distance and copulas) and large-scale significance testing. Briefly, the first work describes the development of clinically relevant single-subject metrics of gene set (pathway) differential expression, N-of-1-\emph{pathways} Mahalanobis distance (MD) scores. Next, the second article describes a method which overcomes a major shortcoming of the MD framework by accounting for inter-gene correlation. Lastly, the statistics developed in the previous works are re-purposed to analyze single-cell RNA-sequencing data derived from rare cells. Importantly, these works represent an interdisciplinary effort and show that creative solutions for pressing issues become possible at the intersection of statistics, biology, medicine, and computer science.

\vspace{2.5pc}

\noindent Statistical Advisor: Walter W. Piegorsch\\
Biomedical Informatics Advisor: Yves A. Lussier

\input{chapter1}
\input{chapter2}
\input{chapter3}
\input{chapter4}
\input{chapter5}

% \input{chapter6}
%\input{chapter7}
%

\setboolean{@twoside}{false}

\printbibliography[heading=refs]
\appendix
\renewcommand{\thetable}{A.\arabic{table}}
\input{appendixA}
\includepdf[pages=-]{md.pdf}
\renewcommand{\thetable}{B.\arabic{table}}
\input{appendixB}
\includepdf[pages=-]{md_supplement.pdf}
\renewcommand{\thetable}{C.\arabic{table}}
\input{appendixC}
\includepdf[pages=-]{clustered_T_smmr_v10.pdf}
\renewcommand{\thetable}{D.\arabic{table}}
\input{appendixD}
\includepdf[pages=-]{clustered_T_suppl_v4.pdf}
\renewcommand{\thetable}{E.\arabic{table}}
\input{appendixE}
\includepdf[pages=-]{ctcs.pdf}
\renewcommand{\thetable}{F.\arabic{table}}
\input{appendixF}
% \includepdf[pages=-, offset=0 -25]{ctcs_supplment.pdf}
\includepdf[pages=-]{ctcs_supplment.pdf}

%\clearpage
% \input{bibliography}
% \bibliography{/Users/grantschissler/Dropbox/bib/default, /Users/grantschissler/Dropbox/bib/library}

%\printbibliography


\end{document}
