%\documentclass[draft]{ua-thesis}
\documentclass[final]{ua-thesis}
\usepackage{multirow}
\usepackage[section] {placeins}
\usepackage{setspace}
%\onehalfspacing
%\doublespacing

% this package allows insertion of entire pdfs
\usepackage[final]{pdfpages}

%\linespread{1.75}
%\addtolength{\footskip}{24 pt}  % pushes page no. down at bottom of page

%\usepackage[breaklinks]{hyperref}
\usepackage{graphicx}
\usepackage{amsmath,amsfonts,amssymb}
\usepackage{amsthm}
%\usepackage{soul}  %for underline characters
%\usepackage{natbib}
%\bibliographystyle{plainnat}
%\bibpunct{(}{)}{;}{a}{,}{,}
\usepackage[backend=biber,style=authoryear,citestyle=authoryear,sorting=none]{biblatex}
\bibliography{/Users/grantschissler/Dropbox/bib/default,/Users/grantschissler/Dropbox/bib/library}
\defbibheading{refs}[refs]{%
\chapter*{References}%
}


\newtheorem{definition}{Definition}[section] %For definition and numbered in sections
\numberwithin{equation}{section}
%\numberwithin{table}{subsection}
%\numberwithin{figure}{subsection}

%\long\def\symbolfootnote[#1]#2{\begingroup% For special symbols in footnote
%\def\thefootnote{\fnsymbol{footnote}}\footnote[#1]{#2}\endgroup}

%\setlength{\topmargin}{0in}
%\setlength{\textheight}{8.5in}
%\setlength{\evensidemargin}{0in}
%\setlength{\oddsidemargin}{0in}
%\setlength{\textwidth}{6.5in}

\newcommand\solidrule[1][1cm]{\rule[0.5ex]{#1}{.4pt}}
\newcommand\dottedrule{\mbox{%
\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}%
\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}\solidrule[.3mm]\hspace{.7mm}}}
\newcommand\dashedrule{\mbox{%
\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}\solidrule[1mm]\hspace{1mm}}}
\newcommand\dasheddotted{\mbox{%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}%
\solidrule[1mm]\hspace{.4mm}\solidrule[.2mm]\hspace{.4mm}}}

\director{Walter W. Piegorsch \& Yves A. Lussier}
\author{A. Grant Schissler}
\title{Statistical and Bioinformatic Contributions to Single-Subject Transcriptome Analytics}
\date{2016}
    \makeindex

%\ifpdf \pdfinfo{ /Author (Qijun Fang) /Title  (Hierarchical Bayesian Benchmark Dose Analysis) } \fi


\begin{document}
\maketitle

\chapter*{Acknowledgments}
First and foremost, thanks go to my advisor Dr. Walter Piegorsch. For what I have learned in wide areas of Statistics through him, his excellent guidance in this project as well as his constant encouragement and support throughout the years. His creative ideas and the time we spent on discussion were absolutely essential to the completion of this project.
%
%Thanks are due my dissertation committee members, Drs. Katherine Y. Barnes, Rabi Bhattacharya, Anton Westveld, D. Dean Billheimer and Chengcheng Hu for their precious suggestions during my research on this project. Special thanks are due Dr. Rabi Bhattacharya for giving me wonderful lectures in advanced theoretic statistics and offering me many other great helps during my doctoral study. Also, thanks are due my colleagues in the University of North Carolina at Wilmington, Drs. Susan J.~Simmons, Cuixian (Tracy) Chen and Yishi Wang for their helpful comments and suggestions during the preparation of the publications regarding to this research. I also want to thank all who have taught me mathematics and statistics through those interesting courses.
%
%Further thanks are due my supervisor for my internship at the Biostatistis and Data Management department (BADM) of the Ventana Medical System, Inc, Dr. James Ranger-Moore for providing me funding for my doctoral study, broadening my view of statistical applications in medical industry and allowing me to use the BADM workstation for computational intensive tasks in this project. Special thanks are due my manager Dr. Isaac Bai for teaching me to use SAS software and sharing with me his experience working in industry. I would also like to thank all my other colleagues in Ventana for it is so pleasant to work with them during the past five years.
%

Finally I would like to thank my wife Fang for her love and unconditional support.

\chapter*{Dedication}
\thispagestyle{topright}
\begin{center}To my wife, my parents, my grandparents and my arriving baby.\end{center}

%\begin{vim_bug_workaround}
%\end{vim_bug_workaround}

\tableofcontents

\listoffigures
\listoftables

\chapter*{Abstract}
\noindent This dissertation serves as a unifying document of three related articles developed during my dissertation research. The projects described involve the development of single-subject transcriptome (i.e. gene expression data) methodology for precision medicine and related applications. While working in this setting, traditional statistical approaches are largely unavailable due to prohibitive sample size and lack of independent replication. This leads one to rely on informatic devices including knowledge base integration (e.g., gene set annotations; Section \ref{sec:nof1pathways}) and external data sources (e.g., estimation of inter-gene correlation; Appendix \ref{App:C}). Common statistical themes include multivariate statistics (such as Mahalanobis distance and copulas) and computational-extensive significance testing. Briefly, the first work\footnote{Appendix \ref{App:A}: Dynamic changes of RNA-sequencing expression for precision medicine: N-of-1-\emph{pathways} Mahalanobis distance within pathways of single subjects predicts breast cancer survival} describes the development of clinically relevant single-subject metrics of gene set (pathway) differential expression, N-of-1-\emph{pathways} Mahalanobis distance (MD) scores. Next, the second article\footnote{Appendix \ref{App:C}: Testing for differentially expressed genetic pathways with single-subject N-of-1 data in the presence of inter-gene correlation} describes a method which overcomes a major shortcoming of the MD framework by accounting for inter-gene correlation. Lastly, the statistics developed in the previous works are re-purposed to analyze single-cell RNA-sequencing data derived from rare cells\footnote{Appendix \ref{App:E}: Analysis of aggregated cell-cell statistical distances within pathways unveils therapeutic-resistance mechanisms in circulating tumor cells}. Importantly, these works represent an interdisciplinary effort and I hope to show that creative solutions for pressing issues become possible at the intersection of statistics, biology, medicine, and computer science.

\vspace{1pc}

\noindent Statistical Advisor: Walter W. Piegorsch\\
Biomedical Informatic Advisor: Yves A. Lussier

\input{chapter1}
\input{chapter2}
\input{chapter3}
\input{chapter4}
\input{chapter5}

% \input{chapter6}
%\input{chapter7}
%

\setboolean{@twoside}{false}

\printbibliography[heading=refs]
\appendix
\renewcommand{\thetable}{A.\arabic{table}}
\input{appendixA}
\includepdf[pages=-]{md.pdf}
\renewcommand{\thetable}{B.\arabic{table}}
\input{appendixB}
\includepdf[pages=-]{md_supplement.pdf}
\renewcommand{\thetable}{C.\arabic{table}}
\input{appendixC}
\includepdf[pages=-]{clustered_T_smmr_v10.pdf}
\renewcommand{\thetable}{D.\arabic{table}}
\input{appendixD}
\includepdf[pages=-]{clustered_T_suppl_v4.pdf}
\renewcommand{\thetable}{E.\arabic{table}}
\input{appendixE}
\includepdf[pages=-]{ctcs.pdf}
\renewcommand{\thetable}{F.\arabic{table}}
\input{appendixF}
% \includepdf[pages=-, offset=0 -25]{ctcs_supplment.pdf}
\includepdf[pages=-]{ctcs_supplment.pdf}

%\clearpage
% \input{bibliography}
% \bibliography{/Users/grantschissler/Dropbox/bib/default, /Users/grantschissler/Dropbox/bib/library}

%\printbibliography


\end{document}
